src/include/sof/list.h

```diff
@@ -72,6 +72,30 @@ static inline void list_item_append(struct list_item *item,
 	list->prev = item;
 }
 
+/* adds item before reference item */
+static inline void list_add_item_before_ref_item(struct list_item *ref_item,
```

Comment:
We should align name with the rest of the list functions. Maybe something like list_item_add_before() and list_item_add_after().

Reply:
Renamed.